### PR TITLE
F-strings docs: fixing link to Format Specifiers

### PR DESCRIPTION
**Pull request by KatherineMichel**
Submitting this PR to amend f-string docs update made by Mariatta: https://github.com/python/cpython/pull/4888/files#r157292842

### CODE REVIEW

File: Doc/reference/lexical_analysis.rst [hunk 1]
```diff
@@ -654,10 +654,11 @@ expression or conversion result.  An empty string is passed when the
 format specifier is omitted.  The formatted result is then included in
 the final value of the whole string.
 
-Top-level format specifiers may include nested replacement fields.
-These nested fields may include their own conversion fields and
-:ref:`format specifiers <formatspec>`, but may not include more
-deeply-nested replacement fields.
+Top-level format specifiers may include nested replacement fields. These nested 
+fields may include their own conversion fields and format specifiers, but may
```
Comment (Mariatta) [inline Doc/reference/lexical_analysis.rst]: This mention of format specifiers should still be linked like before.

**Review by Mariatta — APPROVED** [review]
Thanks! 😄

**Comment by Mariatta** [comment]
Thanks @KatherineMichel 🌮

**Comment by KatherineMichel** [comment]
Woohoo congrats taco! @Mariatta

**Pull request merged**